packet.go

```diff
@@ -88,11 +144,36 @@ func recvPacket(r io.Reader) (uint8, []byte, error) {
 	l, _ := unmarshalUint32(b)
 	b = make([]byte, l)
 	if _, err := io.ReadFull(r, b); err != nil {
+		debug("recv packet %d bytes: err %v", l, err)
 		return 0, nil, err
 	}
+	if debugDumpRxPacketBytes {
+		debug("recv packet: %s %d bytes %x", fxp(b[0]).String(), l, b[1:])
```

Comment:
you don't need the .String(), if fxp(b[0]) implements Stringer then it'll be called anyway